Aflores plymouth (#234)
osisoft/Edge-Data-Store-Docs: content/egress/prepare-egress-destinations.md

uid: PrepareEgressDestinations
---

# Configure egress destinations

AVEVA Data Hub and PI Server destinations require additional configuration to receive OMF messages. Once you configure a destination, you can use the `EndpointId` parameter to specify it as the destination of the data in periodic and manual egress configurations.

## AVEVA Data Hub destinations

> To configure AVEVA Data Hub to receive OMF messages from EDS, create an OMF connection in AVEVA Data Hub. Creating an OMF connection results in an available OMF endpoint that can be used by the EDS egress mechanism.

Reviewer: ```suggestion
To configure AVEVA Data Hub to receive OMF messages from EDS, create an OMF connection in AVEVA Data Hub. Creating an OMF connection results in an available OMF endpoint that EDS can use for the egress mechanism.
```
